packages/block-editor/src/components/media-placeholder/README.md

```diff
@@ -111,13 +119,19 @@ The call back receives an array with the new files. Each element of the collecti
 
 - Type: `Function`
 - Required: Yes
+- Platform: Web | Mobile
+
+The argument of the callback is an object containing the following properties:
```

Comment:
According to the description above, the argument is an array containing those kinds of objects

Reply:
An array with those kinds of objects is an argument in `gallery` case which we are going to adjust in the second iteration. Current PR is a first iteration (simpler one). Now we are investigating possibilities of adding multiple images according to the [comment](https://github.com/wordpress-mobile/gutenberg-mobile/issues/1298#issuecomment-524272903).

Reply:
Oh, I thought it always returned an array on the web, but it seems the argument can be either an array or an object depending on the `multiple` prop. I think the existing documentation was wrong and it should explain this difference in behavior